.github/workflows/rafiki/env-setup/action.yml

```diff
@@ -5,8 +5,6 @@ runs:
   using: "composite"
   steps:
     - uses: pnpm/action-setup@v2
-      with:
-        version: 7
```

Comment:
This can be removed since I've added pnpm's version in the root `package.json`.
https://github.com/interledger/rafiki/blob/4a7201869e345fbb54f9550ee393f2c24581be60/package.json#L7-L11